source/includes/administration/_reports_pricing_v2.md

```diff
@@ -0,0 +1,178 @@
+## Reports
+
+Reports using the pricing v2 engine
+
+<!------------------- GET ORGANIZATION REPORT --------------------->
+
+### Get an organization report
+
+`GET /reports/organization_pricing?organization_id=:id&start_date:=end_date=&end_date=:end_date`
+
+Retrieves a list of usage grouped by currency and containing categories and their products. There is a cost sum at the currency, category and product level of granularity. This will create a report using applied pricings from the closest reseller to the organization.
```

Comment:
Maybe **Generates a report** rather than **This will create a report**

Reply:
6c2e3a9f91f7e73660ad634a3263df65853ce5ee